src/bucket.ts

```diff
@@ -79,6 +79,21 @@ interface WatchAllOptions {
   versions?: boolean;
 }
 
+/**
+ * @typedef {object} AddLifecycleRuleOptions Configuration options for Bucket#addLifecycleRule().
+ * @property {string} [append=true] The new rules will be appended to any
+ *     pre-existing rules.
+ */
+export interface AddLifecycleRuleOptions {
+  append?: boolean;
+}
+
+export type LifecycleRule = {
+  action: {type: string, storageClass?: string}|string;
+  condition: {[key: string]: boolean | Date | number | string};
+  storageClass?: string;
+};
```

Comment:
It would help to call out that `storageClass` is only required when using `setStorageClass` in documentation.

Reply:
New docs: https://github.com/googleapis/nodejs-storage/pull/471/files#r228275474